cloudmock/cloud_mock.go

```diff
@@ -63,15 +65,21 @@ func NewCloudMock() *CloudMock {
 	}
 
 	traceClient := cloudtrace.NewTraceServiceClient(conn)
+	mockTraceClient := mocktrace.NewMockTraceServiceClient(conn)
 	metricClient := monitoring.NewMetricServiceClient(conn)
 	return &CloudMock{
 		conn,
 		grpcServer,
 		traceClient,
+		mockTraceClient,
 		metricClient,
 	}
 }
 
+func (mock *CloudMock) ClientConn() *grpc.ClientConn {
```

Comment:
What is the reason for exposing the underlying connection?

Reply:
The way the Go exporter works for testing is it instantiates the exporter and passes in the client connection info (see [here](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/8c1bbc5eb74bd29b602d739f88e98b185a13fc76/exporter/trace/cloudtrace_test.go#L90)). Right now after we spin up the mock server/clients with the library, there's no way to fetch the client info to pass to the exporter, so I added a getter to fetch this info.